### PR TITLE
K4B-2931 validate cert dates

### PR DESCRIPTION
**Pull request by baronbosse**
Use the correct function to validate cert dates

### CODE REVIEW

File: src/xmerl_dsig.erl [hunk 1]
```diff
@@ -221,19 +220,20 @@ do_find_content([Head | PTail], Parents, Elem) ->
 %% the MinaMeddelanden API.
 %% It will now verify all signatures inside the SOAP structure.
 -spec verify(Element :: #xmlElement{},
-             Fingerprints :: [#'OTPCertificate'{}] | [fingerprint()] | any) ->
-          ok | {error, bad_digest | bad_signature | cert_not_accepted}.
-verify(Element, Fingerprints) ->
```
Comment (kivra-kenand) [inline src/xmerl_dsig.erl:226]: @doc above still reference Fingerprints several times

File: src/xmerl_dsig.erl [hunk 2]
```diff
@@ -221,19 +220,20 @@ do_find_content([Head | PTail], Parents, Elem) ->
...
+             CaCerts :: [#'OTPCertificate'{}] | [binary()] | any) ->
+          ok | {error, bad_digest | bad_signature} |
+          {error, {bad_cert, atom()}}.
```
Comment (kivra-kenand) [inline src/xmerl_dsig.erl]: `{error, {bad_cert, public_key:bad_cert_reason()}}`

**Review by kivra-kenand — APPROVED** [review]

File: src/xmerl_dsig.erl (hunk 2 above)
Comment (baronbosse) [reply src/xmerl_dsig.erl]: I don't think it was exported. Lemme check if it works.

File: src/xmerl_dsig.erl (hunk 1 above)
Comment (baronbosse) [reply src/xmerl_dsig.erl:226]: will fix

File: src/xmerl_dsig.erl (hunk 2 above)
Comment (baronbosse) [reply src/xmerl_dsig.erl]: that worked.

File: src/xmerl_dsig.erl (hunk 1 above)
Comment (baronbosse) [reply src/xmerl_dsig.erl:226]: fixed

**Pull request merged**